charts/akeyless-api-gateway/templates/deployment.yaml

```diff
@@ -1,5 +1,11 @@
 apiVersion: apps/v1
+{{- if eq .Values.kind "Deployment" }}
```

Comment:
Deployment should be default value and change to DaemonSet only if the user asked for it

Reply:
It's the default see the else at the end of the condition